src/extension.ts

```diff
@@ -86,7 +85,8 @@ export async function activateInternal(ctx: vscode.ExtensionContext, perfStats:
         }
         ext.experimentationService = await createExperimentationService(ctx, targetPopulation);
 
-        (new SurveyManager()).activate();
+        // Temporarily disabled
```

Comment:
I assume this is "enable if we need to do any surveys" kind of deal? If so, can you update the comment?

"Temporarily" can mean a lot of things 😆